yammer-shaded/pom.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This pom file should never be released.  This is just the intermediate one needed for the relocation.